src/main/java/com/sfuentes/Student.java

```diff
@@ -0,0 +1,30 @@
+package com.sfuentes;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Student {
+  String[][] students;
+  Map<String, Integer> categories = new HashMap<>();
+  int menCount;
+  int womanCount;
+
+  public Student(String[][] students) {
+    this.students = students;
+  }
+
+  public Map<String, Integer> sortByWomenAndMen(String[][] students) {
```

Comment:
The name of the method says sort... but you are not sorting the data. A more appropriate name could be getCountByGender or something like that.

Reply:
fixed